adr/recipes/file_counts.py

```diff
@@ -0,0 +1,35 @@
+from __future__ import print_function, absolute_import
+
+from collections import defaultdict
+
+from ..query import run_query
+
+def run(args):
+
+    header = ['Revision', 'Total Files', 'Total Covered']
+    covered_files = run_query('covered_files')['data']
+    total_files = run_query('total_files')['data']
+
+    by_revision = {}
+    by_date = {}
+    for item in covered_files:
+        # if we don't have 100 artifacts, something is broken, no data, or still ingesting
+        if item[2] >= 100:
+            # default total files=-1, in some cases this is reported
+            by_revision[item[0]] = {'covered': item[3], 'total': -1}
+            by_date[item[1]] = item[0]
+
+    for item in total_files:
+        if item[0] in by_revision:
+            by_revision[item[0]]['total'] = item[2]
+
+    data = []
+    dates = by_date.keys()
+    dates.sort(reverse=True)
+
+    # For a simple report 10 data points is good enough
+    for date in dates[0:10]:
```

Comment:
You might want to add a --limit (see the task_durations recipe for an example) that defaults to 10. This way you can get a bigger sample if desired.